### PR TITLE
Exclude browser-actions/setup-chrome from renovate

### PR DESCRIPTION
**Pull request by vinzscam**
## Hey, I just made a Pull Request!

Excluding the action from renovate until we can dig more into what is causing the issue.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
Alright! 👍 

Could you reference the actual issue in this PR or a comment in the config file?

File: .github/renovate.json5 [hunk 1]
```diff
@@ -40,6 +40,10 @@
       matchSourceUrls: ['https://github.com/yarnpkg/berry'],
       enabled: false,
     },
+    {
```
Comment (vinzscam) [inline .github/renovate.json5:43]:
```suggestion
    // https://github.com/backstage/backstage/issues/27123
    {
```

**Comment by coreydaley** [comment]
I can't leave a review, but I approve `of` this pull request :)

**Review by freben — APPROVED** [review]
I'll do it on your behalf, @coreydaley !

**Pull request merged**